tests/libmigration/migration.go

```diff
@@ -99,10 +99,27 @@ func RunDecentralizedMigrationAndExpectToCompleteWithDefaultTimeout(virtClient k
 	return RunDecentralizedMigrationAndExpectToComplete(virtClient, sourceMigration, targetMigration, MigrationWaitTime*2)
 }
 
+func CheckSynchronizationAddressPopulated(virtClient kubecli.KubevirtClient, migration *v1.VirtualMachineInstanceMigration) {
+	kv := libkubevirt.GetCurrentKv(virtClient)
+	synchronizationAddress := kv.Status.SynchronizationAddress
+	Expect(synchronizationAddress).ToNot(BeNil())
+
+	Eventually(func() string {
+		migration, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Get(context.Background(), migration.Name, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
Do you want this to fail the whole test or do you want this to be retried? (`g.Expect`)

Reply:
Fail the whole test. The migration resource should be there at this point, if it is not or we are unable to get it, that is a failure of the test.